extensions/wikia/Search/classes/QueryService/Select/AbstractSelect.php

```diff
@@ -197,92 +222,100 @@ public function searchAsApi( $fields = null, $metadata = false, $keyField = null
 			$numPages = $config->getNumPages();
 			$limit = $config->getLimit();
 			$response = [
-					'total' => $total,
-					'batches' => $total > 0 ? $numPages : 0,
-					'currentBatch' => $total > 0 ? $config->getPage() : 0,
-					'next' => 0,
-					'items' => $resultSet->toArray( $fields, $keyField )
-					];
+				'total' => $total,
+				'batches' => $total > 0 ? $numPages : 0,
+				'currentBatch' => $total > 0 ? $config->getPage() : 0,
+				'next' => 0,
+				'items' => $resultSet->toArray( $fields, $keyField )
+			];
 			if ( $total > 0 ) {
-				$response[ 'next' ] = min( [ $numPages * $limit, $config->getStart() + $limit ] ) +
-					$config->mustAddMatchedRecords() + 1;
+				$response['next'] =
+					min( [ $numPages * $limit, $config->getStart() + $limit ] ) + $config->mustAddMatchedRecords() + 1;
 			}
 		} else if ( $fields ) {
 			$response = $resultSet->toArray( $fields, $keyField );
 		} else {
 			$response = $resultSet->toArray( null, $keyField );
 		}
+
 		return $response;
 	}
-	
+
 	/**
-	 * Retrieves an existing match, or forces the child class to retrieve a match. 
-	 * @return \Wikia\Search\Match\Article|\Wikia\Search\Match\Wiki|\Wikia\Search\false|boolean
+	 * Retrieves an existing match, or forces the child class to retrieve a match.
+	 *
+	 * @return \Wikia\Search\Match\Article|\Wikia\Search\Match\Wiki|boolean
 	 */
 	public function getMatch() {
 		$config = $this->getConfig();
 		if ( $config->hasMatch() ) {
 			return $config->getMatch();
 		}
+
 		return $this->extractMatch();
 	}
-	
+
 	/**
-	 * This hook should be overidden by children to access the appropriate kind of match.
-	 * @return NULL
+	 * This hook should be overridden by children to access the appropriate kind of match.
+	 *
+	 * @return null
 	 */
 	protected function extractMatch() {
 		return null;
 	}
-	
+
 	/**
 	 * Creates an instance of Solarium_Query_Select configured by searchconfig.
+	 *
 	 * @return \Solarium_Query_Select
 	 */
 	protected function getSelectQuery() {
 		$query = $this->getClient()->createSelect();
 		$query->setDocumentClass( '\Wikia\Search\Result' );
-		$this->registerQueryParams( $query )
-		     ->registerComponents( $query );
+		$this->registerQueryParams( $query )->registerComponents( $query );
+
 		return $query->setQuery( $this->getQuery() );
 	}
-	
+
 	/**
 	 * This is abstract because the dismax and lucene query services behave differently here.
+	 *
 	 * @return string
 	 */
 	abstract protected function getQuery();
-	
+
 	/**
 	 * Allows us to configure components in child instances.
+	 *
 	 * @param \Solarium_Query_Select $query
+	 *
 	 * @return \Wikia\Search\QueryService\Select\AbstractSelect
 	 */
 	protected function registerComponents( Solarium_Query_Select $query ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$query". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)